### PR TITLE
feat adds beforeLabel and afterLabel props

### PR DESCRIPTION
**Pull request by jonthomp**
## What kind of change does this PR introduce?

Feature - resolves https://github.com/supabase/ui/issues/16

## What is the current behavior?

None

## What is the new behavior?

`beforeLabel` and `afterLabel` props added to all inputs

before_and_after_label_dark
before_and_after_label_light

### CODE REVIEW

File: src/components/Checkbox/Checkbox.module.css [hunk 1]
```diff
@@ -63,6 +63,16 @@
...
 }
 
+.sbui-checkbox__label-container__label__span .sbui-checkbox__label-text-before,
+.sbui-checkbox__label-container__label__span .sbui-checkbox__label-text-after {
+  @apply text-gray-400 dark:text-gray-300;
+}
+
+.sbui-checkbox__label-text-before::after,
+.sbui-checkbox__label-text-after::before {
+  content: ' : ';
```
Comment (MildTomato) [inline src/components/Checkbox/Checkbox.module.css:73]: @jonthomp ah, sorry, I don't think we need these - my bad example in the issue description.
I think I was just using an example from the supabase dashboard but the actual ui library doesn't need to add any `:` characters I don't think

**Review by MildTomato — CHANGES_REQUESTED** [review]
looks great !

File: src/components/Checkbox/Checkbox.module.css (hunk 1 above)
Comment (jonthomp) [reply src/components/Checkbox/Checkbox.module.css]: @MildTomato no problemo, all removed

**Comment by MildTomato** [comment]
🔥  nice!

**Pull request merged**